src/migrations/1640037852136_nft_custody_unanchored.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  /**
+   * This view is identical to `nft_custody` but it will only refreshed during microblock updates
+   * to calculate an unanchored snapshot of NFT custody.
+   */
+  pgm.createMaterializedView('nft_custody_unanchored', {}, `
```

Comment:
I could be misreading -- but where is `nft_custody_unanchored` refreshed?

Reply:
Whoops, good catch. I also found a bug so I had to revisit the refresh logic, check it out.